test/dotnet.Tests/dotnet.Tests.csproj

```diff
@@ -29,7 +29,7 @@
     <Compile Include="..\dotnet-build.Tests\**\*.cs" LinkBase="dotnet-build" />
     <Compile Include="..\dotnet-clean.Tests\**\*.cs" LinkBase="dotnet-clean" />
     <Compile Include="..\dotnet-sdk-check.Tests\**\*.cs" LinkBase="dotnet-sdk-check" />
-    <Compile Include="..\dotnet-format.Tests\**\*.cs" LinkBase="dotnet-format" />
```

Comment:
Under `dotnet-format.Test` folder, there is an existing test file https://github.com/dotnet/sdk/blob/main/test/dotnet-format.Tests/GivenDotnetFsiExecutesAndGeneratesHelpText.cs
So this old config just include that file. I just change it to reflect its current purpose